src/meta-client/src/client.rs

```diff
@@ -216,6 +233,19 @@ pub struct MetaClient {
     store: Option<StoreClient>,
     procedure: Option<ProcedureClient>,
     cluster: Option<ClusterClient>,
+    region_follower: Option<RegionFollowerClientRef>,
+}
+
+pub type RegionFollowerClientRef = Arc<dyn RegionFollowerClient>;
+
+/// A trait for clients that can manage region followers.
+#[async_trait::async_trait]
+pub trait RegionFollowerClient: Sync + Send + Debug {
+    async fn add_region_follower(&self, request: AddRegionFollowerRequest) -> Result<()>;
+
+    async fn remove_region_follower(&self, request: RemoveRegionFollowerRequest) -> Result<()>;
+
+    async fn start(&self, urls: &[&str]) -> Result<()>;
```

Comment:
Do we need a stop method?

Reply:
No, the `start` method of `MetaClient` performs initialization.